lib/solveEquation/stepThrough.js

```diff
@@ -183,11 +221,18 @@ function addSimplificationSteps(steps, equation, debug=false) {
   let oldEquation = equation.clone();
 
   const leftSteps = simplifyExpressionNode(equation.leftNode, false);
+  const leftFactorSteps = factor(equation.leftNode, false);
   const leftSubSteps = [];
   for (let i = 0; i < leftSteps.length; i++) {
     const step = leftSteps[i];
     leftSubSteps.push(EquationStatus.addLeftStep(equation, step));
   }
+
+  for (let i = 0; i < leftFactorSteps.length; i++) {
```

Comment:
I have no idea. I was trying to mimic what was already in place. I'm not sure what the original intentions were.